src/applications/ivc-champva/10-7959C/config/form.js

```diff
@@ -473,7 +474,7 @@ const formConfig = {
       title: 'Form signature',
       pages: {
         formSignature: {
-          path: 'form-signature',
+          path: UPLOADS_COMPLETE_PATH,
```

Comment:
Using the constant since this is the page directly after the `MissingFileOverview`